src/markdown.cpp

```diff
@@ -3365,6 +3365,11 @@ void MarkdownOutlineParser::parseInput(const QCString &fileName,
         {
           docs.prepend("@ianchor " +  generatedId + "\\ilinebr ");
         }
+        else
+        {
+          QCString autoId = AnchorGenerator::instance().generate(title.str());
+          docs.prepend("@ianchor " +  autoId + "\\ilinebr ");
```

Comment:
this misses the `{...}` title argument, and then will not be matched in commentscan.l line 1341.